code/admin/models/log.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
Shouldn't this be logs ?

Reply:
no, delete action happens through singular (form) module. This is correct.